prog/vm/nexus.rb

```diff
@@ -494,14 +545,15 @@ def before_run
         nic.incr_destroy
       end
 
+      vm.vm_storage_volumes.each do |vol|
+        dev = vol.storage_device
+        dev ||= StorageDevice.dataset.where(vm_host_id: vm.vm_host_id, name: "DEFAULT").first
```

Comment:
this line is necessary until all of the VMs are refreshed or they all have the vm_storage_volume <-> storage_device relationship properly setup. I need it for runners, for the long running ones, I will manually set them and then remove this line in an additional PR.